packages/ceramic-cli/src/index.ts

```diff
@@ -103,25 +110,61 @@ program
     }
   })
 
-program
-  .command('change <docId> [content]')
-  .option('--owners <owners>', 'Change owner of this document (only 3ID)')
-  .description('Update the content of a document')
-  .action(async (docId, content, { owners }) => {
-    if (!validateDocId(docId)) {
-      console.error(`Invalid docId: ${docId}`)
-      return
-    }
-    if (typeof content !== 'undefined') content = JSON.parse(content)
-    if (typeof owners === 'string') owners = owners.split(',')
-    const ceramic = new CeramicClient()
-    try {
-      const doc = await ceramic._updateDocument(docId, content, owners)
-      console.log(JSON.stringify(doc.content, null, 2))
-    } catch (e) {
-      console.error(e)
-    }
-    ceramic.close()
-  })
+const pin = program.command('pin')
+pin.description('Ceramic local pinning API')
+
+pin
+    .command('add <docId>')
+    .description('Pin document')
+    .action(async (docId) => {
```

Comment:
Cool, I didn't know you could do it in this way :D

Reply:
yep :D